### ISSUE TITLE
[XPress] WooCommerce styling

### ISSUE THREAD

**Issue by mikecreuzer** [Audentio/xpress-issues#154]
┆Issue is synchronized with this [ClickUp ](https://app.clickup.com/t/guqf9p) by [Unito](https://www.unito.io/learn-more)

**Comment by Ian-Hitt** [comment]
Some suggestions to enhance the experience:

1. Search products component (perhaps replace main search with product search filter on store pages?)
2. Cart dropdown, or main link
3. Add widget locations for product single, product category view, and shopping cart view. Woocommerce adds lots of widgets, but adding theme locations specific to woocmmerce pages probably makes some sense
4. Fix main component styling (buttons, links, headings, maybe block components), but keep simple so that they inherit properly from XF theme
5. Switch out pagination with XF pagination

<img width="498" alt="screen shot 2018-10-31 at 11 56 23 am" src="https://user-images.githubusercontent.com/15836384/47804917-077c6280-dd04-11e8-859f-20141cd3160c.png">

**Comment by RexSeven** [comment]
Check box styling also needs looking at ![alt text](https://fdoc.co.uk/sharex/MwnQlMPAtj.gif)

**Comment by ichpen** [comment]
Ooh is this going to finally get looked at? :)

We need a cart dropdown + cart item counter asap. I've CSS'd my way out of most of the funkiness but a ajaxified cart can't be added that easily.

Also missing/needed standard woocommerce sidebar filters.

**Comment by RexSeven** [comment]
Fingers crossed although I was happy when it was set as priority and we've not had much movement!

**Comment by ichpen** [comment]
Right, I'm hoping @LukasWieditz and @mikecreuzer  and the TH gang simply go through the woocommerce flow and fix up all the issues in one go. There are too many to report literally from css issues, to invalid XF pagination showing up on woocommerce pages, to JS issues, missing sidebars etc etc. I've sort of got it working on my site but far from ideal.

It needs a bit of TLC.

We absolutely need a dedicated cart icon and placement though not just a sub menu item which we can already add ourselves.

Rant over on this topic for the week. :)

**Comment by ichpen** [comment]
Folks been a year since this was marked priority.
